webpack/components/SelectOrg/SetOrganization.js

```diff
@@ -44,6 +44,8 @@ class SetOrganization extends Component {
         pathname: redirectPath,
         state: { orgChanged: this.state.item },
       }));
+
+    window.location.reload();
```

Comment:
I'm seeing this reload necessary to prevent strange behavior between react pages.  @amirfefer let me know if you have any other ideas here.

Reply:
@amirfefer ping

Reply:
It does a full page reload and kills the SPA behavior, is that the only way? it seems the fact that there are few source of truth of the  current organization might be the cause, this solution may work, but it feels like a workaround to me.

Reply:
Given that this is for a release that is coming soon I chose to fix this the quick and dirty way but I'm open to suggestions.

Reply:
Looks like this fixes the issue, and I tend to agree, it’s probably our best option for now. 

could you just add a TODO comment mentioning that this is a workaround ?

Reply:
Sure thing, will add.